auth-web/src/App.vue

```diff
@@ -3,6 +3,7 @@
     <div class="header-group" ref="headerGroup">
       <sbc-loader :show="showLoading" />
       <sbc-header
+        class="flex-direction-column"
```

Comment:
Would one of these work instead?

https://vuetifyjs.com/en/styles/flex/#flex-direction

Reply:
.flex-column

Reply:
let me bookmark this page -^